Common/CustomException/BadRequestException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace CustomException
+{
+    public class BadRequestException : Exception
+    {
+        public BadRequestException(string message)
+            : base(message)
+        { }
+
+        public BadRequestException()
+        { }
+
+        public BadRequestException(string message, Exception inner)
```

Comment:
why do we need this constructor with Exception inner ?

Reply:
Test slack integration

Reply:
what does it mean? I did not understand.

Reply:
Check telegram ))